src/components/NcAppNavigationItem/NcAppNavigationItem.vue

```diff
@@ -493,6 +493,9 @@ export default {
 		undoButtonAriaLabel() {
 			return t('Undo changes')
 		},
+		actionsBoundariesElement() {
+			return document.querySelector('#content-vue')
```

Comment:
What happens if the element is not found? Does it fall back to undefined, hence body in NcActions?

Reply:
I didn't check exactly what happens inside but it works fine with an invalid query selector. I guess you're right, prop is null so the default value is used. I'll check that.

Reply:
@raimund-schluessler Good thing you asked :grin: Default prop value is actually only used if the prop is undefined.

New problem: NcAppNavigationItem's `actionsBoundariesElement` is computed too early so #content-vue does not exist yet.

I didn't face any use case where it is problematic that a nav item action's boundaries are the <body>.
I wonder if specifying the boundaries in this case is really necessary. Do you know that? Or who might know?